CHANGELOG.adoc

```diff
@@ -2,6 +2,14 @@
 
 This file documents all notable changes to https://github.com/devonfw/IDEasy[IDEasy].
 
+== 2024.12.002
+
+Release with new features and bugfixes:
+
+* https://github.com/devonfw/IDEasy/issues/885[#885]: Gcviewer starts in foreground fixed
+
+The full list of changes for this release can be found in https://github.com/devonfw/IDEasy/milestone/15?closed=1[milestone 2024.11.001].
```

Comment:
Thanks for adding the new release here. Was too late yesterday so I haven't done it yet.
Please fix the link as follows:
```suggestion
The full list of changes for this release can be found in https://github.com/devonfw/IDEasy/milestone/17?closed=1[milestone 2024.12.002].
```

Reply:
adjusted the link